src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -38,162 +34,69 @@ NonClientIslandWindow::~NonClientIslandWindow()
 {
 }
 
-// Method Description:
-// - Used to initialize the XAML island for the non-client area. Also calls our
-//   base IslandWindow's Initialize, which will initialize the client XAML
-//   Island.
-void NonClientIslandWindow::Initialize()
-{
-    _nonClientSource = DesktopWindowXamlSource{};
-    auto interop = _nonClientSource.as<IDesktopWindowXamlSourceNative>();
-    winrt::check_hresult(interop->AttachToWindow(_window));
-
-    // stash the child interop handle so we can resize it when the main hwnd is resized
-    interop->get_WindowHandle(&_nonClientInteropWindowHandle);
-
-    _nonClientRootGrid = winrt::Windows::UI::Xaml::Controls::Grid{};
-
-    _nonClientSource.Content(_nonClientRootGrid);
-
-    // Call the IslandWindow Initialize to set up the client xaml island
-    IslandWindow::Initialize();
-}
-
-// Method Description:
-// - Sets the content of the non-client area of our window to the given XAML element.
-// Arguments:
-// - content: a XAML element to use as the content of the titlebar.
-// Return Value:
-// - <none>
-void NonClientIslandWindow::SetNonClientContent(winrt::Windows::UI::Xaml::UIElement content)
-{
-    _nonClientRootGrid.Children().Clear();
-    _nonClientRootGrid.Children().Append(content);
-}
-
+const double XAML_FOCUSRECT_THICKNESS_LEFT = 2;
+const double XAML_MOUSEHOOVER_THICKNESS_LEFT = 1;
+double NonClientIslandWindow::NonClientDragBarWidth = 0.0; // Set by TerminalApp initialization

```

Comment:
This particular variable makes more sense as a member variable of `NonClientIslandWindow` rather than a static variable.